courses/management/commands/create_verified_enrollment.py

```diff
@@ -0,0 +1,161 @@
+        """Handle command execution"""
+        user = fetch_user(options["user"])
+
+        run = CourseRun.objects.filter(courseware_id=options["run"]).first()
+        if run is None:
+            raise CommandError(
+                "Could not find course run with courseware_id={}".format(options["run"])
+            )
+
+        product = Product.objects.filter(
+            object_id=run.id, content_type=ContentType.objects.get_for_model(CourseRun)
+        ).first()
+        if product is None:
+            raise CommandError(
+                "No product found for that course with courseware_id={}".format(
+                    options["run"]
+                )
+            )
+
+        if run.course.is_country_blocked(user):
+            raise CommandError(
+                "Enrollment is blocked of this course with courseware_id={} for user {}".format(
+                    options["run"], options["user"]
+                )
+            )
+
+        # PaidCourseRun should only contain fulfilled or review orders
+        # but in order to avoid false positive passing in order__state__in here
+        paid_course_run = PaidCourseRun.objects.filter(
+            user=user,
+            course_run=run,
+            order__state__in=[Order.STATE.FULFILLED, Order.STATE.REVIEW],
+        ).first()
+        if paid_course_run:
+            raise CommandError(
+                "User {} already enrolled in this course with courseware_id={}\nEnrollment order:{}".format(
+                    options["user"], options["run"], paid_course_run.order
+                )
+            )
+
+        if not run.is_upgradable:
+            raise CommandError(
+                "The course with courseware_id={} is not upgradeable or the upgrade deadline has been passed".format(
+                    options["run"]
+                )
+            )
+
+        discount = Discount.objects.filter(
+            for_flexible_pricing=False, discount_code=options["code"]
+        ).first()
+        if not discount:
+            raise CommandError(
+                "That enrollment code {} does not exist".format(options["code"])
+            )
+
+        if discount.products.exists() and not (
+            discount.products.filter(product=product).exists()
+        ):
+            raise CommandError(
+                "That enrollment code {} is invalid for course with courseware_id={}".format(
+                    options["code"], options["run"]
+                )
+            )
+
+        if not discount.check_validity(user):
+            raise CommandError(
+                "That enrollment code {} for course with courseware_id={} is invalid for user {}".format(
+                    options["code"], options["run"], options["user"]
+                )
+            )
+
+        discounted_price = DiscountType.get_discounted_price([discount], product)
+
+        if discounted_price > 0:
+            raise CommandError("Discount code is not 100% off")
```

Comment:
This will keep it standardized with the naming we are using above. We're using the Enrollment code everywhere I think.

```suggestion
            raise CommandError("Enrollment code is not 100% off")
```